nb-adapters/adapter-s4j/src/main/java/io/nosqlbench/adapter/s4j/dispensers/MessageConsumerOpDispenser.java

```diff
@@ -109,28 +111,33 @@ public MessageConsumerOpDispenser(DriverAdapter adapter,
             "consumer.deadLetterPolicy",
             "consumer.negativeAckRedeliveryBackoff",
             "consumer.ackTimeoutRedeliveryBackoff"};
+
         HashMap<String, String> stmtLvlConsumerConfRawMap = new HashMap<>();
         for (String confKey : stmtLvlConsumerConfKeyNameList ) {
             String confVal = parsedOp.getStaticConfigOr(confKey, "");
             stmtLvlConsumerConfRawMap.put(
                 StringUtils.substringAfter(confKey, "consumer."),
                 confVal);
         }
-
-        this.combinedS4jConfigObjMap.putAll(
-            s4jSpace.getS4JClientConf().mergeExtraConsumerConfig(stmtLvlConsumerConfRawMap));
```

Comment:
There is some weird configuration happening here because it's JMS with added Pulsar configuration